ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -3066,6 +3067,42 @@ def test_torch_instance_bitwise_or(
     )
 
 
+# bitwise_or_
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="bitwise_or_",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
```

Comment:
and then you could define this as `available_dtypes=helpers.get_dtypes("valid"),` because the filtering of the dtypes will be handled by the function decorator above.

Reply:
😎 Okay, thanks I'm going to fix it. Thanks

Reply:
@AnnaTz , Hey could u review this?